daemon/SecretsImpl/pluginfunctionwrappers.cpp

```diff
@@ -18,6 +18,28 @@ PluginState Daemon::ApiImpl::pluginState(PluginBase *plugin)
     return PluginState(plugin->isAvailable(), plugin->isLocked());
 }
 
+FoundLockedResult Daemon::ApiImpl::queryLockSpecificPlugin(
+        const QMap<QString, Sailfish::Secrets::EncryptionPlugin*> &encryptionPlugins,
+        const QMap<QString, StoragePluginWrapper*> &storagePlugins,
+        const QMap<QString, EncryptedStoragePluginWrapper*> &encryptedStoragePlugins,
+        const QString &lockCodeTarget)
+{
+    bool found = true;
+    bool locked = true;
+    Result result(Result::Succeeded);
+    if (PluginBase *p = storagePlugins.value(lockCodeTarget)) {
```

Comment:
Order ways defined and only one can be target then could be like:
```
    if (PluginBase *p = storagePlugins.value(lockCodeTarget) ||
            p = encryptedStoragePlugins.value(lockCodeTarget) ||
            p = encryptionPlugins.value(lockCodeTarget)) {
        locked = p->supportsLocking() && p->isLocked();
    } else {
        found = false;
    }
```

Reply:
good point, will update to this form